corehq/apps/es/tests/__init__.py

```diff
@@ -4,3 +4,9 @@
 from .test_filters import *
 from .test_queries import *
 from .test_case_search_es import *
+from corehq.apps.es.case_search import flatten_result
+
+
+__test__ = {
+    'flatten_result': flatten_result
+}
```

Comment:
There is a test for this already, here: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/es/tests/test_case_search_es.py#L144-L156, you can remove that one if you think the doctest is clearer.